scripts/ci/gitlab/pipeline/publish.yml

```
@@ -214,3 +214,23 @@ publish-rustdoc:
     - echo "___Rustdoc was successfully published to https://paritytech.github.io/polkadot/___"
   after_script:
     - rm -rf .git/ ./*
+
+# Ref: https://github.com/paritytech/opstooling/issues/111
+update-node-template:
+  stage:                           publish
+  extends:                         .kubernetes-env
```

```suggestion
  extends:                         .kubernetes-env
  variables:
    GIT_STRATEGY:          none
```
It seems that from polkadot repo on;y tag is used. Maybe we don't need to fetch repo?